inst/Upgrades.qml

```diff
@@ -0,0 +1,533 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades 
+{
```

Comment:
this line is not matched with a closing bracket, so just put `}` at the end of the file. When I do it the Upgrader works for me, so hopefully that will fix it for you as well.

In general I would recommend you to style the indexing as in e.g., https://github.com/jasp-stats/jaspRegression/blob/master/inst/Upgrades.qml, it's then easier to spot mistakes

Reply:
Thanks again!